### ISSUE TITLE
Copy and paste block causes "Opps, don't know how to load this file!"

### ISSUE THREAD

**Issue by microbit-carlos** [microsoft/pxt-microbit#4944]
Edit: This happens when there is an image in the clipboard and a user copy/pastes a block in the workspace.
https://github.com/microsoft/pxt-microbit/issues/4944#issuecomment-1431216892

-----

**Describe the bug**

I had this occur a couple of times in the past, but it went away fairly quickly (not sure if a number of refreshes or something else fixed it).

Today it seems to occur more consistently, at least on my system, also when using an incognito window. When trying to copy and paste a block with the keyboard it shows this warning.

<img width="402" alt="Screenshot 2023-02-15 at 10 54 29" src="https://user-images.githubusercontent.com/29712657/219008258-20b77946-0290-40eb-841e-cbd03d774382.png">

**To Reproduce**

This video shows a MakeCode session with the dbg flag, where I simply select a block and press Ctrl+C and Ctrl+V:

https://user-images.githubusercontent.com/29712657/219008747-45748649-a2dd-4fe7-83ad-261675c8b85c.mp4

<br>
**Expected behavior**
The warning not to appear.

**Screenshots**

Included above.

**micro:bit version (please complete the following information):**

N/A

**Desktop (please complete the following information):**
 - OS: macOS 12 Monterey
 - Browser: Chrome
 - Version: 109

**Smartphone (please complete the following information):**
N/A

**Additional context**

makecode.microbit.org version:  [5.0.12](https://github.com/Microsoft/pxt-microbit/releases/tag/v5.0.12)
Microsoft MakeCode version:  [8.0.7](https://github.com/Microsoft/pxt/releases/tag/v8.0.7)
microbit runtime version:  [v2.2.0-rc6](https://github.com/lancaster-university/microbit/releases/tag/v2.2.0-rc6)
codal-microbit-v2 runtime version:  [v0.2.40](https://github.com/lancaster-university/codal-microbit-v2/releases/tag/v0.2.40)

**Comment by microbit-carlos** [comment]
@JohnVidler had a great epiphany about checking my clipboard and I can confirm that this issue is only replicate for me when I have an image (in this case a screenshot) in my clipboard.

**Comment by abchatra** [comment]
We don't support multiple things in clipboard as we can't load other files

**Comment by Jaqster** [comment]
@microbit-carlos - I can't reproduce this bug.  Just successfully Ctrl+C, Ctrl+V pasted several blocks in the workspace.  Can you provide more detailed repro steps?

**Comment by microbit-carlos** [comment]
Yes, sorry, I should have added the https://github.com/microsoft/pxt-microbit/issues/4944#issuecomment-1431216892 comment to the original message. To trigger the issue you need to have an image in your clipboard.

**Comment by abchatra** [comment]
May be mac specific. Doesn't repro in the windows machine.

**Comment by abchatra** [comment]
@carlosperate could reproduce in Mac.

**Comment by microbit-matt-hillsdon** [comment]
We also saw this when working on keyboard navigation and I can confirm the reproductions above. It's very confusing as it just appears to be random whether it happens from a user perspective as they're unlikely to link the error to the contents of their clipboard as they are unlikely to appreciate that copying a block didn't put it on their clipboard as such.

If you have a hex file on your clipboard and copy paste a block then the block will be pasted and then hex file will load replacing your project.

One option is to clear the system clipboard on copy - PR with that approach here: https://github.com/microsoft/pxt/pull/10463

(I think the Blockly team may also explore using the system clipboard for copy/paste as part of considering keyboard nav because of the awkwardness of the keyboard nav plugin and apps that implement cross-tab paste both wanting to override clipboard actions.)
